packages/shared/src/components/feeds/HorizontalFeed.tsx

```diff
@@ -33,7 +33,7 @@ export default function HorizontalFeed<T>({
   useEffect(() => {
     const element = feedContainerRef;
     if (!element) {
-      return null;
+      return () => {};
```

Comment:
But this is the cleanup effect right?
Shouldn't it be something like `return false` or `return undefined` now? *(CC @capJavert you might know)

Reply:
I think it might work, but it's a quick/dirty fix.
We probably want to revert this to a opposite solution saying "if (element)" do something instead?

Reply:
> We probably want to revert this to a opposite solution saying "if (element)" do something instead

I also think this would be best, and the return function would be consistent on all returns this way as well.

Reply:
We would still have to return an empty function but on the other side of the condition, which is at the last part of the function

Reply:
This is only linter complaining for `consistent-return` rule, React does not care really, I would go with `return undefined` yes 👍

Reply:
It was actually React that throws the error 😓 

But yeah, after testing to return `undefined` it work normally. Let me update everything quickly 🥳 

![Screenshot 2024-07-02 at 3 48 31 AM](https://github.com/dailydotdev/apps/assets/13744167/9dbc3116-c565-4dc0-b29d-b1a15c7ab645)

Reply:
This happens if you return anything but a function, if you `return undefined` its like doing just `return` but our linter does not scream.